tracker/src/Tracker/Models/ExecutedOrders.hs

```diff
@@ -0,0 +1,123 @@
+module Tracker.Models.ExecutedOrders where
+
+import ErgoDex.Amm.Orders
+import ErgoDex.Contracts.Types as Currencies
+import ErgoDex.State
+import ErgoDex.Amm.Pool
+import ErgoDex.Class
+import ErgoDex.Types
+
+import Explorer.Class
+
+import CardanoTx.Models
+
+import           Data.Aeson (FromJSON, ToJSON)
+import qualified PlutusTx.Prelude as P    
+import           Ledger           
+import           GHC.Generics                (Generic)
+import           RIO
+
+data ExecutedSwap = ExecutedSwap
+  { config       :: Swap
```

Comment:
you have the common pattern for all Executed(Swap/Deposit/Redeem): config, orderInputId, userOutputId, poolOutputId, poolInputId. Why not to create smth like ExecutedOrder a?

Reply:
done